### PR TITLE
Improve Status & HeartBeatBar Styling & Animation

### PR DESCRIPTION
**Pull request by raphaelbernhart**
# Description

This PR fixes some styling issues regarding the Status and HeartBeatBar. For example the status titles where not really centered. In this course I also updated the HeartBeatBar Animation. It now is smoother, more elegant and better for the UX.

## Type of change

- User interface (UI)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

#### Before:
Bildschirmfoto 2022-01-21 um 15 38 49

#### Now:
Bildschirmfoto 2022-01-21 um 17 03 22

https://user-images.githubusercontent.com/48283236/150564588-5d5a4fd5-a26a-4f7a-92f5-ed1b9a282e5e.mov

### CODE REVIEW

File: src/components/HeartbeatBar.vue [hunk 1]
```diff
@@ -200,11 +226,22 @@ export default {
...
+            width: 5px;
+        }
+
         &:not(.empty):hover {
-            transition: all ease-in-out 0.15s;
+            transition: all ease 0.15s;
             opacity: 0.8;
             transform: scale(var(--hover-scale));
         }
+        &.active-sibling {
```
Comment (Saibamen) [inline src/components/HeartbeatBar.vue:241]:
```suggestion
        }

        &.active-sibling {
```

File: src/components/HeartbeatBar.vue [hunk 2]
```diff
@@ -168,9 +172,30 @@ export default {
 
         getBeatTitle(beat) {
             return `${this.$root.datetime(beat.time)}` + ((beat.msg) ? ` - ${beat.msg}` : ``);
+        },
+        // Toggling the activeSibling class on hover over the current hover item
```
Comment (Saibamen) [inline src/components/HeartbeatBar.vue:176]:
```suggestion
        },

        // Toggling the activeSibling class on hover over the current hover item
```

File: src/components/HeartbeatBar.vue [hunk 3]
```diff
@@ -168,9 +172,30 @@ export default {
...
+            // Return if the hovered element has empty class
+            if (element.classList.contains("empty")) {
+                return;
+            }
+
+            // Check if Previous Sibling is heartbar element and doesn't have the empty class
+            if (previous.children && !previous.classList.contains("empty")) {
+                previous.classList.toggle("active-sibling");
+            }
+            // Check if Next Sibling is heartbar element and doesn't have the empty class
```
Comment (Saibamen) [inline src/components/HeartbeatBar.vue:192]:
```suggestion
            }

            // Check if Next Sibling is heartbar element and doesn't have the empty class
```

**Review by Saibamen — APPROVED** [review]

**Comment by raphaelbernhart** [comment]
@Saibamen How long do you think this takes to merge because it's just a simple UI optimization? I am just asking if it's worth it to further work on this project for me.

**Comment by Saibamen** [comment]
Don't know. Ask the author -> @louislam 

But active development is very low in comparison to 2 months ago. @louislam: Why? You don't have time now or what?

**Review by Computroniks — APPROVED** [review]

**Comment by louislam** [comment]
Thanks, it is btf.👍

**Comment by louislam** [comment]
Unfortunately, it is breaking some animation. It has been reverted.

**Pull request merged**